frontend/src/app/pages/marker-page/marker-page.component.ts

```diff
@@ -98,7 +98,7 @@ export class MarkerPageComponent implements OnInit {
                 this.lastSliceID = slice.index;
             }
             this.marker.feedData(slice);
-            if (slice.isLastInBatch()) {
+            if (slice.isLastInBatch() || (slice.index === this.scan.numberOfSlices - 1)) {
```

Comment:
I think that `isLastInBatch()` should be enough. Otherwise, maybe there is an error on the backend side?

Backend returns information about SliceIndex and indicator which SliceIndex is last in current batch. Checking manually for last Slice in Scan should be redundant.

Reply:
Batch size is fixed to 10 slices (`validation-page.component.ts`, line 23). Therefore, for Scans with <10 Slices, last Slice index is always set to 9. `isLastInBatch()` compares all of the Slices looking for one with id 9, which it won't find in that case.

Reply:
Ohhh... Ok. So as I thought - Backend implementation is broken ;)

[This part](https://github.com/jpowie01/MedTagger/blob/master/backend/medtagger/api/scans/service_web_socket.py#L29-L32) should be rewritten to something like this:

```python
slices = list(business.get_slices_for_scan(scan_id, begin, count, orientation=orientation))
slices_to_send = list(reversed(slices)) if reversed_order else slices
last_in_batch = begin if reversed_order else begin + len(slices_to_send) - 1
for index, (_slice, image) in enumerate(slices_to_send):
    ...
```

BTW. I would like to refactor fetching images from Cassandra to use batch query in the future, so above code will probably change. For now, it'll be fine.